addons/im_livechat/static/src/legacy/public_livechat.scss

```diff
@@ -23,6 +23,11 @@ $o-mail-thread-window-zindex: $zindex-modal + 1 !default;
         width: 100%;
         height: 100%!important;
         box-shadow: none;
+
+        .o_frontend_to_backend_nav:not(.d-none) ~ & .o_thread_window_title {
```

Comment:
I would replace `.o_thread_window_title` by `.o_thread_window_header`.

This gives the same visual, but it feels more natural that the backend_nav and the chat window header are overlapping, so we put some padding left to offset content inside the header.
With `.o_thread_window_title`, it's specific to an element of the chat window, and it assumes it always exists and it's always the 1st element. It's true, but it's much easier to not have to know all of this to understand the implication of this rule.

Reply:
Quite right, good shout !
I've made the change ;)